src/ansi-c/ansi_c_language.cpp

```diff
@@ -75,6 +75,7 @@ bool ansi_c_languaget::parse(
   ansi_c_parser.in=&codestr;
   ansi_c_parser.set_message_handler(get_message_handler());
   ansi_c_parser.for_has_scope=config.ansi_c.for_has_scope;
+  ansi_c_parser.Floatn=config.ansi_c.Floatn;
```

Comment:
How about `ansi_c.ts_18661_3_fp_types` or something, as `Floatn` is not very descriptive, and while a mouthful that at least gives you something to google

Reply:
ok, renamed to ts_18661_3_Floatn_types